hw3/index.js

```diff
@@ -0,0 +1,47 @@
+class Vector {
+    constructor(arr) {
+        this.coordinates = arr;
+    }
+}
+
+Vector.prototype.add = function(vector) {
```

Comment:
@matveyprupas Why do you use `Vector.prototype` contruction to add a method?

Reply:
Because I mixed up ES5 and ES6 syntax. Fixed it. Wrote everything in ES6 syntax.